src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/TryConvertTool.cs

```diff
@@ -97,6 +101,6 @@ public Task<bool> RunAsync(IUpgradeContext context, IProject project, Cancellati
             return null;
         }
 
-        private string GetArguments(IProject project) => string.Format(CultureInfo.InvariantCulture, TryConvertArgumentsFormat, Path, GetMSBuildPath(), project.Required().FileInfo);
+        private string GetArguments(IProject project) => string.Format(CultureInfo.InvariantCulture, TryConvertArgumentsFormat, Path, GetMSBuildPath(), project.Required().FileInfo) + _tryConvertOptions;
```

Comment:
Nit-pick: Since we already have `TryConvertArgumentsFormat`, it might be better to just update that to have a spot for additional options and pass `_tryConvertOptions` into the `string.Format` call.